### DISCUSSION TITLE
Logflare exposed without much notice in docs when self hosting, and how would I add basic auth to it?

### DISCUSSION THREAD

**Discussion by marpe** [supabase/supabase#34978] [Feature Requests]
# Improve documentation

This is as much a question as a suggestion for improvement. I am not familiar with neither supabase nor digital ocean where I am "self hosting" a supabase instance. I just recently started to poke around for learning purposes.

## Link 

Link to [documentation](https://supabase.com/docs/guides/self-hosting/docker#securing-your-services)

## Describe the problem

I used the droplet template provided by digital ocean to spin up a supabase instance and link a domain to the studio. I find it a bit odd that the logflare instance isn't behind basic auth like the dashboard by default, isn't this a potential security risk?

I tried adding
```yml
    plugins:
      - name: cors
      - name: basic-auth
        config:
          hide_credentials: true
```
to the `supabase/docker/volumes/api/kong.yml` file but it didn't seem to have an effect, the `:4000` endpoint is still totally public

## Describe the improvement

I'd expect at least a notice that the service is exposed, and suggestions on how to secure it.

**Comment by Hallidayo** [comment]
Hi everyone, due to inactivity on this issue I've moved the issue over to discussions/enhancements.

**Comment by aantti** [comment]
This was indeed an oversight for too long - apologies. Addressed via PR #42857 and will also consider adding a how-to on how to re-enable external access to Logflare in case there's a need to ingest logs from outside of a [self-hosted Supabase](https://supabase.com/docs/guides/self-hosting) setup.

Separately - I don't think DigitalOcean's Supabase droplet, or even Terraform to deploy self-hosted Supabase have been maintained on the DO's side for the past couple of years or more. Official upstream configuration is [here](https://github.com/supabase/supabase/tree/master/docker).
